src/main/java/uk/gov/hmcts/cmc/claimstore/services/ClaimService.java

```diff
@@ -232,27 +227,74 @@ public Claim saveClaim(
             .build();
 
         Claim savedClaim = caseRepository.saveClaim(user, claim);
+        String pin = pinResponse.map(GeneratePinResponse::getPin).orElse(null);
+        createClaimEvent(authorisation, user, pin, savedClaim);
+        trackClaimIssued(savedClaim.getReferenceNumber(), savedClaim.getClaimData().isClaimantRepresented());
+
+        return savedClaim;
+    }
+
+    @LogExecutionTime
+    @Transactional(transactionManager = "transactionManager")
+    public Claim saveLegalRepClaim(
+        String submitterId,
+        ClaimData claimData,
+        String authorisation
+    ) {
+        String externalId = claimData.getExternalId().toString();
+        User user = isExistingClaim(authorisation, externalId);
+
+        Optional<GeneratePinResponse> pinResponse = getPinResponse(claimData, authorisation);
+        Optional<String> letterHolderId = pinResponse.map(GeneratePinResponse::getUserId);
+        String submitterEmail = user.getUserDetails().getEmail();
+
+        Claim claim = Claim.builder()
+            .claimData(claimData)
+            .submitterId(submitterId)
+            .externalId(externalId)
+            .submitterEmail(submitterEmail)
+            .createdAt(nowInLocalZone())
+            .letterHolderId(letterHolderId.orElse(null))
+            .claimSubmissionOperationIndicators(ClaimSubmissionOperationIndicators.builder().build())
+            .build();
+
+        Claim savedClaim = caseRepository.saveClaim(user, claim);
+        String pin = pinResponse.map(GeneratePinResponse::getPin).orElse(null);
+        createClaimEvent(authorisation, user, pin, savedClaim);
+        trackClaimIssued(savedClaim.getReferenceNumber(), savedClaim.getClaimData().isClaimantRepresented());
+
+        return savedClaim;
+    }
+
+    private User isExistingClaim(String authorisation, String externalId) {
+        User user = userService.getUser(authorisation);
+
+        caseRepository.getClaimByExternalId(externalId, user).ifPresent(claim -> {
+            throw new ConflictException(
```

Comment:
This is not new code that is added as part of this PR, it's existing code in master, it's just extracted out to a method. From what i understand, this can be removed if CCD returns 409 in case of conflict as you say. @sabahirfan what do you suggest we do as part of this ticket?

Reply:
I agree that we should not clean up as part of this PR, but we should not introduce more tech debt as well. There was no need to extract this as a separate method and use in new method where it's unnecessary.

Reply:
Thanks @simad I have reverted this extraction and it's usage

Reply:
so you are saying, we don't need to throw conflict exception in saveRepresentedClaim method but keep code as in master to just throw exception in saveClaim method?

Reply:
Cheers @sabahirfan , i see that you have done it now.